### ISSUE TITLE
Null value cannot be deserialized to an array.

### ISSUE THREAD

**Issue by hez2010** [Tornhoof/SpanJson#118]
```csharp
byte[] a = SpanJson.JsonSerializer.Generic.Utf8.Deserialize(Encoding.UTF8.GetBytes("null"));
```

it will throw an exception.

**Comment by Tornhoof** [comment]
Thank you for your bug report. I can confirm this bug.

**Comment by Tornhoof** [comment]
Fixed in 2.0.9
